backend/app/services/knowledge/folder_service.py

```
@@ -416,6 +417,58 @@ def move_document(
+            try:
+                KnowledgeFolderService.move_document(
+                    db=db,
+                    document_id=doc_id,
+                    folder_id=folder_id,
+                    user_id=user_id,
+                )
+                success_count += 1
+            except ValueError:
+                failed_ids.append(doc_id)
+                db.rollback()
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)